x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/rule_management/bulk_actions/bulk_actions_route.schema.yaml

```diff
@@ -31,6 +43,90 @@ paths:
                 - $ref: '#/components/schemas/BulkDuplicateRules'
                 - $ref: '#/components/schemas/BulkManualRuleRun'
                 - $ref: '#/components/schemas/BulkEditRules'
+            examples:
+              example1:
+                summary: Enable all rules with the test tag
+                description: The following request activates all rules with the test tag
+                value:
+                  query: 'alert.attributes.tags: "test"'
+                  action: 'enable'
+              example2:
+                summary: Enable a specific rule by ID
+                description: The following request enables the rule with the specified ID.
+                value:
+                  action: 'enable'
+                  ids:
+                    - '748694f0-6977-4ea5-8384-cd2e39730779'
+              example3:
+                summary: Disable a specific rule by ID
+                description: The following request disables the rule with the specified ID.
+                value:
+                  action: 'disable'
+                  ids:
+                    - '748694f0-6977-4ea5-8384-cd2e39730779'
+              example4:
+                summary: Add tags to rules
+                description: The following request adds tags tag-1 and tag-2 to the rules that have the IDs sent in the payload
+                value:
+                  ids:
+                    - '8bc7dad0-9320-11ec-9265-8b772383a08d'
+                    - '8e5c1a40-9320-11ec-9265-8b772383a08d'
+                  action: 'edit'
+                  edit:
+                    - type: 'add_tags'
+                      value:
+                        - 'tag-1'
+                        - 'tag-2'
+              example5:
+                summary: Dry run - Validate add_index_patterns bulk action
+                description: The following request will validate that the add_index_patterns bulk action can be successfully applied to three rules. The dry_run parameter is specified in query parameters, e.g. POST api/detection_engine/rules/_bulk_action?dry_run=true
+                value:
+                  action: 'edit'
+                  edit:
+                    - value:
+                        - 'test-*'
+                      type: 'add_index_patterns'
+                  ids:
+                    - '81aa0480-06af-11ed-94fb-dd1a0597d8d2'
+                    - 'dc015d10-0831-11ed-ac8b-05a222bd8d4a'
+                    - 'de8f5af0-0831-11ed-ac8b-05a222bd8d4a'
```

Comment:
These are all awesome examples, makes it so much more clear how to use this endpoint!